starter-code/js/index.js

```diff
@@ -1,52 +1,78 @@
-function deleteItem(e){
+function updatePriceByProduct(productPrice, index) {}
 
-}
+var calculatePriceBtn = document.querySelector(".btn-success");
 
-function getPriceByProduct(itemNode){
+calculatePriceBtn.onclick = function getTotalPrice() {
+  var textField = document.querySelector(".quantity");
 
-}
+  var checkout = document.querySelectorAll(".unit-price");
+  var unitQty = document.querySelectorAll(".quantity");
+  var unitPrice = document.querySelectorAll(".unit-price");
+  var total = document.querySelectorAll(".total");
+  var checkoutPrice = document.querySelector("checkout-price");
 
-function updatePriceByProduct(productPrice, index){
+  checkout.forEach(function(oneItem, index) {
+    // console.log(unitPrice[index].innerHTML);
 
-}
+    unitQty[index].innerHTML = textField.value;
+    // console.log(unitQty[index].value);
 
-function getTotalPrice() {
+    var totalPrice = unitPrice[index].innerHTML * unitQty[index].value;
+    total[index].innerHTML = totalPrice;
+    // console.log(totalPrice);
 
-}
+    checkoutPrice = checkoutPrice + totalPrice;
+    checkoutPrice.innerHTML = checkoutPrice;
+  });
```

Comment:
Here you missed a variable that is stocking the total price. It should be declared outside the forEach loop but you're almost there. 
I send you a solution that could make you understand what to do: 
```
  // Defining the function, will be called upon clicking on a button
  calculatePrice = function() {

    // Saving all the elements we're going to use inside variables, to make them 'accessible' inside Javascript
    var totalPrices = document.querySelectorAll(".total-price");
    var quantities = document.querySelectorAll(".unit-quantity");
    var unitPrices = document.querySelectorAll(".unit-price");
    var globalPrice = document.querySelector(".global-price");

    // Creating a counter that we'll use to calculate the big, global price
    var counter = 0;
    
    // Making a loop, so I can calculate each line's price individually
    for(var i=0; i<unitPrices.length; i++) {
      totalPrices[i].innerHTML = Number(quantities[i].value) * Number(unitPrices[i].innerHTML);
      counter += Number(totalPrices[i].innerHTML);
    }

    // Setting the value of the global price DOM element to the counter's value.
    // As a reminder, the counter is the place we stored the sum of all the item's total prices, during the loop
    globalPrice.innerHTML = counter;
    
  };
  
  // Saving the calculate button inside a variable so I can put an 'onclick' on it
  var calculateBtn = document.querySelector(".btn-success");

  // Setting the calculate button's onclick function
  calculateBtn.onclick = calculatePrice;
  ```